### PR TITLE
adding student desk group methods -group 6

### PR DESCRIPTION
**Pull request by NithilaSadheesh**
leave all classroom changes
added methods to assign student to group number and find all student names from group number

### CODE REVIEW

File: StudentDeskGroup.java [hunk 1]
```diff
@@ -18,8 +18,74 @@ public class StudentDeskGroup extends Actor
      * Act - do whatever the StudentDeskGroup wants to do. This method is called whenever
      * the 'Act' or 'Run' button gets pressed in the environment.
      */
+    public void assignSeatsToDeskGroups() {
+        // Getting all student desks
+        List<StudentDesk> allDesks = getWorld().getObjects(StudentDesk.class);
+        for (StudentDesk desk : allDesks) {
+            // checking coordinates to find right desk group
+            if (desk.getY() > 5 && desk.getX() < 4) {
+                desk.setDeskGroup(1);
```
Comment (anirudhb) [inline StudentDeskGroup.java:27]: `Student` doesn't have `setDeskGroup`

**Review by anirudhb — CHANGES_REQUESTED** [review]
Does this compile?

File: StudentDeskGroup.java (hunk 1 above)
Comment (NithilaSadheesh) [reply StudentDeskGroup.java:27]: setDeskGroup is a public method in StudentDesk.java and we are accessing it here

**Review by anirudhb — DISMISSED** [review]
lgtm

**Review by anirudhb — APPROVED** [review]

**Pull request merged**